src/lib/connectors/bookshelf/BillingInvoice.js

```diff
@@ -15,5 +15,9 @@ module.exports = bookshelf.model('BillingInvoice', {
 
   billingInvoiceLicences () {
     return this.hasMany('BillingInvoiceLicence', 'billing_invoice_id', 'billing_invoice_id');
+  },
+
+  linkedBillingInvoices () {
+    return this.hasMany('BillingInvoice', 'original_billing_invoice_id', 'original_billing_invoice_id');
```

Comment:
Forgive my pedantry - I think this comes down to my lack of experience with Bookshelf, but

This suggests that an invoice can have numerous `original_billing_invoice_id`s

What I expected: An invoice can have one `original_billing_invoice_id`, rather than many. 

So I think my question is: Is this intentionally `hasMany`?

Reply:
There is a single `original_billing_invoice_id` column, but this function here is describing the relationship.

For any one of the 3 invoices (original, reversal, rebill), this relationship points from one to all 3 which allows all the linked invoices to be retrieved with a Bookshelf query. 

Don't know if that explains it?